client/command/loot.go

```diff
@@ -0,0 +1,500 @@
+package command
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+	"text/tabwriter"
+	"unicode/utf8"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/bishopfox/sliver/client/spin"
+	"github.com/bishopfox/sliver/protobuf/clientpb"
+	"github.com/bishopfox/sliver/protobuf/commonpb"
+	"github.com/bishopfox/sliver/protobuf/rpcpb"
+	"github.com/bishopfox/sliver/protobuf/sliverpb"
+	"github.com/bishopfox/sliver/util/encoders"
+	"github.com/desertbit/grumble"
+)
+
+var (
+	ErrInvalidLootType = errors.New("invalid loot type")
+)
+
+func lootRoot(ctx *grumble.Context, rpc rpcpb.SliverRPCClient) {
+	filter := ctx.Flags.String("filter")
+	var allLoot *clientpb.AllLoot
+	var err error
+	if filter == "" {
+		allLoot, err = rpc.LootAll(context.Background(), &commonpb.Empty{})
+		if err != nil {
+			fmt.Printf(Warn+"Failed to fetch loot %s\n", err)
+			return
+		}
+	} else {
+		lootType, err := lootTypeFromHumanStr(filter)
+		if err != nil {
+			fmt.Printf(Warn + "Invalid loot type see --help")
+			return
+		}
+		allLoot, err = rpc.LootAllOf(context.Background(), &clientpb.Loot{Type: lootType})
+		if err != nil {
+			fmt.Printf(Warn+"Failed to fetch loot %s\n", err)
+			return
+		}
+	}
+	displayLootTable(allLoot)
+}
+
+func lootAddLocal(ctx *grumble.Context, rpc rpcpb.SliverRPCClient) {
+	localPath := ctx.Args.String("path")
+	if localPath == "" {
+		fmt.Printf(Warn + "Missing local path argument, see --help\n")
+		return
+	}
```

Comment:
Not necessary, since `ctx.Args.String("path")` is a mandatory argument, grumble will error out before you reach that part of the code.